src/apps/experimental/AppLayout.tsx

```diff
@@ -59,40 +60,47 @@ const AppLayout = () => {
                     />
                 </AppBar>
             </ElevationScroll>
+            {/*
+                An extra toolbar is needed here without fixed positioning to take up space at the top of the page and
+                ensure other elements flow properly instead of being covered by the fixed AppBar.
+                https://mui.com/material-ui/react-app-bar/#fixed-placement
+             */}
+            <Toolbar variant='dense' />
```

Comment:
We have handled this with padding in the style overrides. There are some complications since some pages have the standard toolbar, some have the toolbar + tabs, and some have content rendered behind the toolbar (mainly the item details page).

Reply:
Ah that makes sense if there are places where there are things intentionally rendered in that space. I reached for this because I've frequently run into problems trying to account for space of fixed elements, and it's often easiest to reason about it if the space reservation is controlled by the fixed element or a wrapper, or at least is handled in the same place.

Regardless, I ended up reverting this file back to what's in master since none of the other changes here were necessary with more recent changes that made the drawer static.